src/ace/managers/viewport/tilebuffer.c

```diff
@@ -219,6 +229,12 @@ void tileBufferReset(
 		}
 	}
 
+	// Init tile offset lookup table
+	pManager->pTileSetOffsets = memAllocFastClear(sizeof(APTR) * 0xFF);
+	for (UBYTE i = 0; i < 0xFF; ++i) {
```

Comment:
ditto - perhaps put the upper limit in some kind of constant and also add ACE_DEBUG-only checks if tile indices later on are smaller?